# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/html/semantics/interactive-elements/the-dialog-element/resources/submit.jpg
# Commit: 4e18b24fdab6
# Full Hash: 4e18b24fdab6df9bcab9a89f14b9758df5d73281
# Author: sefeng <sefeng@mozilla.com>
# Date: 2020-05-12 04:04:10
# Regressor Bug: 1557303
# File Overlap Count: 1
# Description:
#   Bug 1557303 - Add tests to test form dialog submission r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D73365
# ==============================================================================

diff -r b9e87fde31f7 -r 4e18b24fdab6 testing/web-platform/tests/html/semantics/interactive-elements/the-dialog-element/resources/submit.jpg
Binary file testing/web-platform/tests/html/semantics/interactive-elements/the-dialog-element/resources/submit.jpg has changed
